### ISSUE TITLE
Debug mode

### ISSUE THREAD

**Issue by Alexis-Jacob** [Deepomatic/deepocli#19]
It would be nice if we could activate a debug mode such as 
```python
if os.getenv('debug'):
    import requests
    import logging
    logging.basicConfig(level=logging.DEBUG)
```

**Comment by thomas-riccardi** [comment]
maybe call that `verbose`.

**Comment by maingoh** [comment]
Also we should:
- use `logger = logging.getLogger(__name__)`
- call `logging.basicConfig()` only once in the init of the package

To allow for debug or any other log level I think it is better to use `logging.basicConfig(level=os.getenv('LOG_LEVEL', 'INFO'))` like in `deepomatic-api`

**Comment by maingoh** [comment]
So here we just need to choose what we want to log in debug and what we want to log in info. The rest has been done. @Alexis-Jacob any memory of what you wanted to ignore ?

**Comment by Alexis-Jacob** [comment]
No, my purpose was to see the requests done by deepocli a runtime, but nothing to ignore is occuring to my mind

**Comment by maingoh** [comment]
Alright, we can also put this kind of log in debug https://github.com/Deepomatic/deepocli/blob/103ba48991ff446a0235f0ddb33c08957f001c1f/deepomatic/cli/output_data.py#L26
